docs/fides/docs/guides/extending_data_use.md

```diff
@@ -0,0 +1,40 @@
+# Extending Data Use
+
+
+This page provides an overview of extending the [Data Use taxonomy](/fides/language/taxonomy/data_uses) for your organization. We recommend extending from the existing categories to ensure interopability inside and outside your organization.
+
+One of the core reasons for doing this is adding attribution to be used when exporting data from Fides and adding context/clarity for legal teams to use.
```

Comment:
`doing this` is likely not helpful in communicating what is going on here, be more explicit